### PR TITLE
chkname.c, pwck.c, useradd.c, usermod.c, newusers.c: Allow names that…

### LINKED ISSUE #121: --force-badname
Hello!

After reading [this mailing list thread](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=42874) I'm not sure whether it was intentionally removed, but is there a runtime option or compiler flag that I can use to force [`is_valid_name()`](https://github.com/shadow-maint/shadow/blob/6fbc11ce2178205968c37853db752729359c9893/libmisc/chkname.c#L49-L71) to always `return true`? If not, would you be open to having it added?

Thanks for your work on this project!

### PR DESCRIPTION
**Pull request by edneville**
Allow names that do not conform to standards. Fixes #121

### CODE REVIEW

**Comment by glensc** [comment]
You should generally stick to the same indenting style used in the rest of the files.

**Comment by edneville** [comment]
Thank you @glensc, modified the branch to match what is nearby.

File: src/newusers.c [hunk 1]
```diff
@@ -610,9 +610,9 @@ static void process_flags (int argc, char **argv)
 #endif
 	                         long_options, NULL)) != -1) {
 		switch (c) {
-        case 'b':
-            allow_bad_names = true;
-            break;
+			case 'b':
```
Comment (tpimh) [inline src/newusers.c]: Extra tab here?

Comment (edneville) [reply src/newusers.c]: @tpimh, you're correct. I've pushed another update.

**Pull request merged**